test/tracing/frcnn/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+project(test_frcnn_tracing)
+
+find_package(Torch REQUIRED)
```

Comment:
This is ok for now as you are directly using the CMake from torchvision for the test.

For the future, could you think on how would be a good way for users to rely on torchvision's CMake file in their own CMake, so that it could work as an example for users on how to integrate torchvision in their own projects?

Reply:
Right now I'm having some compilation issues due to some errors of versions of glibc, so most probably we need to include the torchvision cmake reference here